HLS_land_tiles.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import fiona
+import json
+import requests
+import xmltodict
+
+from shapely.geometry import Polygon
+
+
+class hls_land_tiles:
```

Comment:
This should really be in PascalCase

```suggestion
class HLSLandTiles:
```